packages/repack/src/commands/bundle.ts

```diff
@@ -1,10 +1,11 @@
-import { Config } from '@react-native-community/cli-types';
-import webpack from 'webpack';
-import { VERBOSE_ENV_KEY } from '../env';
-import { BundleArguments, CliOptions } from '../types';
-import { loadWebpackConfig } from '../webpack/loadWebpackConfig';
-import { getWebpackEnvOptions } from '../webpack/utils';
-import { getWebpackConfigPath } from './utils/getWebpackConfigPath';
+import { Config } from "@react-native-community/cli-types";
+import webpack from "webpack";
+import { VERBOSE_ENV_KEY } from "../env";
+import { BundleArguments, CliOptions } from "../types";
+import { loadWebpackConfig } from "../webpack/loadWebpackConfig";
+import { getWebpackEnvOptions } from "../webpack/utils";
+import { getWebpackConfigPath } from "./utils/getWebpackConfigPath";
+import Os from "os";
```

Comment:
It looks like the reset of the project uses single quotes, so it seems like a bad idea to change them here.